skynet/env.py

```diff
@@ -6,6 +6,17 @@
 
 is_mac = sys.platform == 'darwin'
 
+
+# utilities
+def tobool(val: str | None):
```

Comment:
you might wanna consider moving this to https://github.com/jitsi/skynet/blob/master/skynet/utils.py